src/Validator/EditorValidator.sol

```diff
@@ -65,4 +62,27 @@ contract EditorValidator is IValidator {
         }
         return true;
     }
+
+    function getRemainEditsCount(address _user) external view returns (uint256) {
+        uint32[5] memory userEdits = edits[_user];
+        uint256 count = 0;
+        for (uint256 i = 0; i < userEdits.length; ) {

```

Comment:
this is a view so no need to gas optimize unchecked since its gonna be free and there is no risk of gas cancelation bcs its gonna go through 5 numbers. You can also simplify  the for to just checking the  else condition


 for (uint256 i = 0; i < userEdits.length; ++i) {
     if (block.timestamp - userEdits[i] >= 1 days) {
        ++count;
     }
 }

Reply:
The github action seems to be failing when I removed the unchecked from the code because the gass was much

![Screenshot from 2022-06-08 13-09-15](https://user-images.githubusercontent.com/41796074/172612990-46bede56-1b5a-4a74-9d0b-6ce117537635.png)

Reply:
thats bcs you need to do another forge snapshot and push it to the repo 👍🏻